src/main/java/com/haruatari/akane/server/kernel/bencode/encode/NumberToken.java

```diff
@@ -0,0 +1,17 @@
+package com.haruatari.akane.server.kernel.bencode.encode;
+
+public class NumberToken implements EncodableToken {
+
+    private int value;
+
+    public NumberToken(int value) {
+        this.value = value;
+    }
+
+
+    @Override
+    public String encode() {
+        String enCodeValue = "i" + value + "e";
```

Comment:
The `enCodeValue` variable is redundant here. you can return the expression directly.